ios-12-1-dep-12-0-arm64.cmake

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2017-2018, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_12_1_DEP_12_0_ARM64_CMAKE_)
+  return()
+else()
+  set(POLLY_IOS_12_1_DEP_12_0_ARM64_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(IOS_SDK_VERSION 12.1)
+set(IOS_DEPLOYMENT_SDK_VERSION 12.0)
+
+set(POLLY_XCODE_COMPILER "clang")
+polly_init(
+    "iOS ${IOS_SDK_VERSION} / Deployment ${IOS_DEPLOYMENT_SDK_VERSION} / Universal (arm64) / \
+${POLLY_XCODE_COMPILER} / \
+c++17 support"
+    "Xcode"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+include(polly_fatal_error)
+
+# Fix try_compile
+include(polly_ios_bundle_identifier)
+
+set(CMAKE_MACOSX_BUNDLE YES)
+set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "iPhone Developer")
+
+set(IPHONEOS_ARCHS arm64)
+set(IPHONESIMULATOR_ARCHS "")
+
+include("${CMAKE_CURRENT_LIST_DIR}/compiler/xcode.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/os/iphone.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx17.cmake")
```

Comment:
Please switch to `cxx14` (see `ios-12-1-dep-11-0-arm64`) or add `cxx17` to the name of toolchain (see `ios-11-3-dep-9-0-device-bitcode-cxx17`).